openmetadata-ui/src/main/resources/ui/src/components/containers/PageLayoutV1.tsx

```diff
@@ -45,14 +48,21 @@ const PageLayoutV1: FC<PageLayoutProp> = ({
         </Col>
       )}
       <Col
-        className="page-layout-v1-center page-layout-v1-vertical-scroll"
+        className={classNames(
+          'page-layout-v1-center page-layout-v1-vertical-scroll',
+          {
+            'flex justify-center': center,
+          }
+        )}
         flex={
           leftPanel && rightPanel
             ? 'calc(100% - 568px)'
             : leftPanel || rightPanel
             ? 'calc(100% - 284px)'
             : '100%'
-        }>
+        }
+        offset={center ? 3 : 0}
+        span={center ? 18 : 24}>
```

Comment:
this change will impact all the pages please test all the pages before merging in, to avoid regression!